code/obj/medicaldiagnosis.dm

```diff
@@ -0,0 +1,68 @@
+				if(do_after(user, (user.traitHolder.hasTrait("training_medical") ? 2 SECONDS : 4 SECONDS)) && !(user.getStatusDuration("stunned") || user.getStatusDuration("weakened") || user.getStatusDuration("paralysis") > 0 || !isalive(user) || user.restrained()))
+					if(isalive(M) == false)
+						boutput(user, "<span style=\"color:red\">You hear nothing inside [(user != M) ? "[M]'s" : "your (AHEM WHAT THE FUCK)"] lungs.</span>")
+						//playsound
+					else if(M.find_ailment_by_type(/datum/ailment/disease/respiratory_failure))
+						boutput(user, "<span style=\"color:red\">You hear fluid sloughing around inside [(user != M) ? "[M]'s" : "your"] lungs, interspersed with crackling noises.</span>")
+						//playsound
+					else if(M.reagents.has_reagent("sarin") || M.reagents.has_reagent("strychnine")) // Are they poisoned with stuff that causes bronchospasm (ADD MORE THINGS YOU NERD)
+						boutput(user, "<span style=\"color:red\">You hear what sounds like a distorted, high-pitched wheeze inside [(user != M) ? "[M]'s" : "your"] lungs.</span>")
+						//playsound
+					else if(ceil(M.get_oxygen_deprivation()) > 80) // pick from 3 awful breath sounds when I feel like adding them
+						boutput(user, "<span style=\"color:red\">You hear what sounds like twitchy, labored breathing interspersed with short gasps inside [(user != M) ? "[M]'s" : "your"] lungs.</span>") // OH BOY THIS IS BAD
+						//playsound
+					else if((M.organHolder.left_lung && M.organHolder.left_lung.get_damage() > 50) || (M.organHolder.right_lung && M.organHolder.right_lung.get_damage() > 50))
+						boutput(user, "<span style=\"color:red\">You hear coarse crackling noises inside [(user != M) ? "[M]'s" : "your"] lungs.</span>")
+						//playsound
+					else if(M.reagents.has_reagent("histamine")) // Are they poisoned with stuff that causes swelling (ADD MORE THINGS YOU NERD)
+						boutput(user, "<span style=\"color:red\">You hear what sounds like low-pitched wheezing inside [(user != M) ? "[M]'s" : "your"] lungs.</span>")
+						//playsound
+					else if(M.reagents.has_reagent("krokodil") || M.reagents.has_reagent("morphine") || M.reagents.has_reagent("haloperidol") || M.reagents.has_reagent("ether") || (M.reagents.has_reagent("ethanol") && M.reagents.get_reagent_amount("ethanol") > 25))
+						boutput(user, "<span style=\"color:blue\">You hear very slow, shallow breathing inside [(user != M) ? "[M]'s" : "your"] lungs.</span>")
+						//playsound
+					else if(M.reagents.has_reagent("cannabidiol") || M.reagents.has_reagent("antihistamine") || (M.reagents.has_reagent("ethanol") && M.reagents.get_reagent_amount("ethanol") < 25))
+						boutput(user, "<span style=\"color:blue\">You hear slightly slow breathing inside [(user != M) ? "[M]'s" : "your"] lungs.</span>")
+						//playsound
+					else if((20 < ceil(M.get_oxygen_deprivation())) && (ceil(M.get_oxygen_deprivation()) < 80)) // pick from 3 stressed breath sounds when I feel like adding them
```

Comment:
Any particular reason you're using ceil here?

Reply:
Not really? It probably made sense to half-lucid me a few nights ago but
I’ll cut it out tomorrow because it seems really unneeded now that I think
about it

Reply:
resolved in next commit